recipes/pygeoogc/.ipynb_checkpoints/meta-checkpoint.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   noarch: python
-  number: 0
```

Comment:
this is in a weird folder

Reply:
This was for the previous package that I removed and replaced with this new one. So it's deleted and is not in this commit.

Reply:
@scopatz Is there anything that I need to do?

Reply:
This ipynb folder shouldn't exist here at all.  Please just delete it

Reply:
I always have it in gitignore so I keep forgetting to check it. I just removed it.